eclair-core/src/main/scala/fr/acinq/eclair/api/JsonSerializers.scala

```diff
@@ -115,3 +115,10 @@ class ThrowableSerializer extends CustomSerializer[Throwable](format => ({ null
 class FailureMessageSerializer extends CustomSerializer[FailureMessage](format => ({ null }, {
   case m: FailureMessage => JString(m.message)
 }))
+
+class NodeAddressSerializer extends CustomSerializer[NodeAddress](format => ({ null},{
+  case IPv4(a, p) => JString(s"${a.getHostAddress}:$p")
```

Comment:
For at least `IPv4` and `IPv6` I'd rather use guava's `HostAndPort` (like we did in `InetSocketAddressSerializer`) which was done for that purpose.